src/constants/relationship_status.js

```diff
@@ -0,0 +1,8 @@
+const RELATIONSHIP_STATUS = {
+  BLANK: '',
+  ACTIVE: 'self',
```

Comment:
A bit confused over the choice and wording of key:value pairs. Shouldn't we just use `active` for `ACTIVE` and `archive` for `ARCHIVE`?

Also think we could use `other` for `OTHER` as a relationship status rather than `etc`/`ETC`.